docs/book/src/topics/managedcluster.md

```diff
@@ -285,6 +285,27 @@ Current limitations
 - Only supports Azure Active Directory Managed by Azure.
   - We will not support Legacy Azure Active Directory
 
+### AKS Cluster Managed Identity
```

Comment:
@nojnhuh with your recent streamlining of AKS feature docs is there a better way to document this?

Reply:
I tried to establish some amount of a pattern in #3550 for what to include in the godoc comments for the type definitions. My initial sense is that what's included here overlaps with the godoc comments enough to justify not needing a separate section here. Definitely no hard rules though, so feel free to add anything here you think is valuable. I'd at least include the AKS doc link in the godoc in any case.

Reply:
godocs are generated after merge to main?

Reply:
The deploy preview for this PR has the generated version of these changes: https://deploy-preview-3425--kubernetes-sigs-cluster-api-provider-azure.netlify.app/reference/v1beta1-api.html#infrastructure.cluster.x-k8s.io/v1beta1.IdentityProfile

Reply:
Love it, will remove the add'l matter I added to the markdown.